norona_marynicole/index.html

```diff
@@ -0,0 +1,184 @@
+      <div>
+        <p>Comment:</p>
+        <textarea name="gcomment" id="gcomment" cols="30" rows="10"></textarea>
+      </div>
+      <div>
+        <button type="button" id="comment_button">COMMENT</button>
+        <button type="button" id="sort_asc" onclick="sortComments('asc')">
+          Sort by Date (Ascending)
+        </button>
+        <button type="button" id="sort_desc" onclick="sortComments('desc')">
+          Sort by Date (Descending)
+        </button>
+      </div>
+    </form>
+
+    <div>
+      <h2 class="opinion">MY OWN OPINION ON F2F OR ONLINE CLASS</h2>
+      <p>
+        In my perspective, conducting a face-to-face class last April 20, 2024,
+        was more convenient for understanding the lesson easily. It was
+        advantageous since not all students have necessary materials like
+        laptops, PCs, and internet access at home, especially for subjects like
+        coding. Face-to-face classes facilitate better understanding of
+        practical activities and provide necessary materials like PCs and
+        internet access. However, in deciding between face-to-face or online
+        classes, safety should be a priority, especially considering the current
+        high-heat index situation.
+      </p>
+    </div>
+
+    <div>
+      <h2 class="title">TESTIMONIALS</h2>
+      <ul>
+        <li>
+          I couldn't ask for a better group mate in coding. Her commitment to
+          teamwork and willingness to lend a hand whenever needed have been
+          instrumental in our success. -Bergado, Shamma
+        </li>
+
+        <li>
+          I heartily endorse Mary Nicole. She is an amazing developer and
+          coworker because of her strong work ethic, collaborative spirit, and
+          technical ability. -Reyes, John
+        </li>
+
+        <li>
+          Coding alongside my group mates has been truly inspiring. Their
+          passion for problem-solving and dedication to excellence make every
+          session rewarding. -Reyes, Paul
+        </li>
+
+        <li>
+          Mary, my classmate, demonstrates a good grasp of coding, though she's
+          not an expert. I've seen her proficiency in various languages and her
+          confidence in solving coding challenges. While she's not at a
+          professional level yet, her dedication and competence shine in our
+          class projects, making her a valuable team member. -Morcoso, Daisy
+        </li>
+      </ul>
+    </div>
+
+    <div>
+      <h2 class="title">FORM</h2>
+      <div>
+        <h2>Contact Me</h2>
+
+        <form class="form">
+          <input type="email" name="email" id="email" placeholder="Email" />
+          <input
+            type="text"
+            name="full_name"
+            id="full_name"
+            placeholder="Full Name"
+          />
+          <input
+            type="text"
+            name="subject"
+            id="subject"
+            placeholder="Subject"
+          />
```

Comment:
Remove unnecessary breaking of lines

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27